README.md

```diff
@@ -38,6 +38,10 @@ Alternatively add symlinks in your `.git/hooks` directory to any of the githooks
 We use a custom Rust compiler configuration conditional to guard the bench mark code. To run the
 bench marks use: `RUSTFLAGS='--cfg=bench' cargo +nightly bench --features=recovery`.
 
+### A note on the `insecure-erase` feature
+
+The `insecure-erase` feature is provided to assist other libraries in building secure secret erasure. When the `insecure-erase` feature is enabled, secret types (`SecretKey`, `KeyPair`, `SharedSecret`, `Scalar`, and `DisplaySecret`) have a method called `insecure_erase` that *attempts* to overwrite the contained secret. This library makes no guarantees about the security of using `insecure_erase`. In particular, since all of these types are `Copy`, the compiler is free to move and copy them, thereby making additional copies in memory. For more information, consult the [`zeroize`](https://docs.rs/zeroize) documentation.
```

Comment:
> since all of these types are `Copy`, the compiler is free to move and copy them

This is misleading, even non-`Copy` types can be copied by the compiler. `Copy` literally means "doesn't run destructors", nothing else.

It should say something like "In particular, the compiler doesn't know that this is a secret so it may arbitrarily copy it anywhere it pleases."

Reply:
Great point. Reworded!